controllers/admin_ctrl.js

```diff
@@ -0,0 +1,93 @@
+var Admin = require("../models/admin");
+var User = require("../models/user");
+const jwt = require('jsonwebtoken');
+const bcrypt = require("bcryptjs");
+const config = require("../config/config");
+
+function loginAdmin(req, res, next) {
+    console.log(req.headers);
+    Admin.findOne({
+      email: req.body.email
+    }, (err, admin) => {
+      if (err) throw err;
```

Comment:
if there is an error you should return 400 bad request to the user. Also if there is no admin returned this doesn't mean that he is unauthorized it should be 404 admin not found. status key is misspelled and no need to return errors you can just return body:'Not found'